# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/gfxVRExternal.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/vr/gfxVRExternal.cpp
--- a/gfx/vr/gfxVRExternal.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/vr/gfxVRExternal.cpp	Fri May 24 11:32:54 2019 +0000
@@ -8,8 +8,8 @@
 
 #include "prlink.h"
 #include "prenv.h"
-#include "gfxPrefs.h"
 #include "mozilla/Preferences.h"
+#include "mozilla/StaticPrefs.h"
 
 #include "mozilla/gfx/Quaternion.h"
 
@@ -526,7 +526,7 @@
 
 #elif defined(XP_WIN)
   if (mShmemFile == NULL) {
-    if (gfxPrefs::VRProcessEnabled()) {
+    if (StaticPrefs::VRProcessEnabled()) {
       mShmemFile =
           CreateFileMappingA(INVALID_HANDLE_VALUE, NULL, PAGE_READWRITE, 0,
                              sizeof(VRExternalShmem), kShmemName);
@@ -624,11 +624,11 @@
     VRExternalShmem* aAPIShmem /* = nullptr*/) {
   MOZ_ASSERT(NS_IsMainThread());
 
-  if (!gfxPrefs::VREnabled()) {
+  if (!StaticPrefs::VREnabled()) {
     return nullptr;
   }
 
-  if ((!gfxPrefs::VRExternalEnabled() && aAPIShmem == nullptr)
+  if ((!StaticPrefs::VRExternalEnabled() && aAPIShmem == nullptr)
 #if defined(XP_WIN)
       || !XRE_IsGPUProcess()
 #endif